modules/appeals_api/spec/factories/notice_of_disagreements.rb

```diff
@@ -17,17 +17,6 @@
     trait :status_received do
       status { 'submitted' }
     end
-    trait :status_completed do
-      status { AppealsApi::NoticeOfDisagreement::COMPLETE_STATUSES.sample }
-    end
-    trait :completed_a_day_ago do
-      updated_at { 1.day.ago }
```

Comment:
Trying to change `updated_at` in this way doesn't actually work, and there were workarounds for it in every spec that used the traits. There weren't very many of them, so I just removed the traits.